.github/workflows/main.yml

```diff
@@ -135,8 +139,7 @@ jobs:
     strategy:
       matrix:
         os:
-          - ubuntu-18.04
-          - ubuntu-20.04
+          - ubuntu-latest
```

Comment:
FYI: at least for Ubuntu the idea was to rely on LTS releases and not on the latest.